### DISCUSSION TITLE
stories_things/basic_content/const/

### DISCUSSION THREAD

**Discussion by giscus** [Light-City/light-comments#4] [Announcements]
# stories_things/basic_content/const/

Stories About C Plus Plus

https://light-city.github.io/stories_things/basic_content/const/

**Comment by BAI-HANK** [comment]
写的很详细,作者用心的传达知识

**Reply by Light-City** [reply]
感谢

**Comment by LeePBlue** [comment]
可以的，清楚了很多

**Comment by coolhuhu** [comment]
在 “7.类中使用const” 一节中，显示有错误，应该是排版的问题？我在github上看是没问题的。

**Comment by lanqingxi** [comment]
Apple::add那里是否应该修改为void类型？保持int类型的话，函数没有返回值会出现warning.

**Comment by ccnuu** [comment]
```cpp
int Apple::getCount() const
{
    take(1);
//    add(); //error
    return apple_number;
}
```
getCount() 成员函数中调用的add()函数没有声明吧? 

头文件声明的两个add成员函数都是有一个入参的
```cpp
int add(int num);
int add(int num) const;
```
应该是没有const修饰的add成员函数没有入参吧
